pkg/apimachinery/identity/context.go

```diff
@@ -121,13 +141,20 @@ var serviceIdentityPermissions = getWildcardPermissions(
 	"serviceaccounts:read", // serviceaccounts.ActionRead,
 )
 
-var serviceIdentityTokenPermissions = getTokenPermissions(
-	"folder.grafana.app",
-	"dashboard.grafana.app",
-	"secret.grafana.app",
-	"query.grafana.app",
-	"iam.grafana.app",
-)
+var serviceIdentityTokenPermissions = func() []string {
+	perm := getTokenPermissions(
```

Comment:
Let's remove `getTokenPermissions`, it adds an unnecessary layer of indirection. Wdyt?